components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/util/TemplateManagerHelper.java

```diff
@@ -208,6 +160,10 @@ public static void validateTemplateGroup(TemplateGroup templateGroup) throws Tem
                 throw new TemplateManagerException("Invalid TemplateGroup configuration file found - UUID is null for" +
                         " templateGroup " + templateGroup.getName());
             }
+            if (templateGroup.getUuid().isEmpty()) {
+                throw new TemplateManagerException("Invalid TemplateGroup configuration file found - UUID cannot be " +
+                        "null" + " for" + " templateGroup " + templateGroup.getName());
+            }
             if (templateGroup.getRuleTemplates().size() == 0) {
                 throw new TemplateManagerException("Invalid TemplateGroup configuration file found - No ruleTemplate" +
```

Comment:
Can we give more context if we can like name of the config file etc

Reply:
In this level, we can only provide the template Group name since we are only passing the templateGroup as a param.